front-end/peer-prep/src/components/Auth/SignInPage/index.tsx

```diff
@@ -25,6 +30,31 @@ const SignInPage = () => {
     )
 }
 
+const googleProvider = new GoogleAuthProvider();
+
+const googleSignIn = () => signInWithPopup(auth, googleProvider)
+  .then((result) => {
+    // This gives you a Google Access Token. You can use it to access the Google API.
+    const credential = GoogleAuthProvider.credentialFromResult(result);
+    const token = credential?.accessToken;
+
+    // The signed-in user info.x
+    const user = result.user;
+    // IdP data available using getAdditionalUserInfo(result)
+    // ...
+
```

Comment:
I would suggest if you could immediately allow the user to navigate to `/home`!

An example could be:
`const googleSignIn = () => signInWithPopup(....)
    .then((result) => {
        useNavigate("/home")
    });
`

Reply:
Thanks for the suggestion, I have fixed it. Let me know if it works now!

Reply:
Yeap it works thanks for the change!